game/PsyDoom/Launcher/Launcher_Tab_Multiplayer.cpp

```diff
@@ -0,0 +1,128 @@
+//------------------------------------------------------------------------------------------------------------------------------------------
+// Setup code and event handling logic for the 'Multplayer' tab
+//------------------------------------------------------------------------------------------------------------------------------------------
+#if PSYDOOM_LAUNCHER
+
+#include "Asserts.h"
+#include "Launcher_Context.h"
+#include "Launcher_Utils.h"
+#include "PsyDoom/Config/Config.h"
+#include "PsyDoom/Config/ConfigSerialization_Multiplayer.h"
+
+BEGIN_DISABLE_HEADER_WARNINGS
+    #include <FL/Fl_Box.H>
+    #include <FL/Fl_Group.H>
+    #include <FL/Fl_Native_File_Chooser.H>
+END_DISABLE_HEADER_WARNINGS
+
+BEGIN_NAMESPACE(Launcher)
+
+//------------------------------------------------------------------------------------------------------------------------------------------
+// Makes the 'co-op' section
+//------------------------------------------------------------------------------------------------------------------------------------------
+static void makeCoopSection(const int x, const int y) noexcept {
+    // Container frame
+    new Fl_Box(FL_NO_BOX, x, y, 336, 30, "Cooperative");
+    new Fl_Box(FL_THIN_DOWN_BOX, x, y + 30, 336, 160, "");
+    
+        // Friendly fire toggle
+    {
```

Comment:
Minor: make sure the comments are indented according to the indentation of the scope they are in.

Reply:
651ba2ce50d4707b3afc846ac310fd167c030567